### PR TITLE
feat: add region:visibility-all shortcut (ctrl+h)

### PR DESCRIPTION
**Pull request by LouisJULIEN**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [X] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_
No shortcut existed to hide all regions. It had to be done through a click tot he dedicated button on the sidebar

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_
by default, ctrl+h now hide all regions if one is visible. If none is visible, it show all regions.

#### What is the current behavior?
_(if this is a breaking or feature change)_
ctrl + h has no behavior on label studio.

#### What libraries were added/updated?
_(list all with version changes)_
None

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_
No

#### Does this change affect security?
_(if so describe the impacts positive or negative)_
No

#### What alternative approaches were there?
_(briefly list any if applicable)_
Click on the sidebar button

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_
None

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [X] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit
Does manual testing counts for a shortcut (I kind of hate myself for asking)?

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

**Comment by LouisJULIEN** [comment]
I reverted my changes made to web/dist, no idea if that's what's expected

**Comment by LouisJULIEN** [comment]
Hello! Would it be possible to review this PR? I tried to make it as small and as little as possible to ease review. Thanks!

**Comment by LouisJULIEN** [comment]
Up! @hlomzik maybe?

**Comment by LouisJULIEN** [comment]
Up!  I'm trying with @yyassi-heartex this time

**Comment by hlomzik** [comment]
/jira create
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10995205925)
> Jira issue [TRIAG-873](https://humansignal.atlassian.net/browse/TRIAG-873) is created

[TRIAG-873]: https://humansignal.atlassian.net/browse/TRIAG-873?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by hlomzik** [comment]
Oh, thank you for patience and for this contribution, @LouisJULIEN !
It's very straightforward and precise improvement, we'll run it through our process and will merge soon!

**Comment by LouisJULIEN** [comment]
Hey @hlomzik , thanks for running the CI. Is everything good?

**Comment by LouisJULIEN** [comment]
up @hlomzik

File: web/libs/editor/src/core/settings/keymap.json [hunk 1]
```diff
@@ -80,6 +80,11 @@
     "key": "alt+h",
     "description": "Toggle selected region visibility"
   },
+  "region:visibility-all": {
+    "key": "ctrl+h",
+    "mac": "command+h",
```
Comment (hlomzik) [inline web/libs/editor/src/core/settings/keymap.json]:
```suggestion
    "mac": "ctrl+h",
```
`⌘+H` is a Mac global hotkey to hide a window, so let's use `ctrl+h` on every platform and explicitly state this in here.

**Review by hlomzik — CHANGES_REQUESTED** [review]

File: web/libs/editor/src/core/settings/keymap.json (hunk 1 above)
Comment (LouisJULIEN) [reply web/libs/editor/src/core/settings/keymap.json]: good catch

**Comment by makseq** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/11183415956)
> Successfully merged: Already up to date.

**Review by hlomzik — APPROVED** [review]

**Review by bmartel — APPROVED** [review]

**Comment by bmartel** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/11236310358)
> Successfully merged: Already up to date.

**Pull request merged**